### ISSUE TITLE
Doesn't recognize @layer at-rule

### ISSUE THREAD

**Issue by jpnelson** [microsoft/typescript-styled-plugin#166]
eg. this:

```
const class = css`
    @layer component {
        margin: 0;
    }
`;
```

Results in this warning: `Unknown at rule @layerts-styled-plugin(9999)`

**Comment by jpnelson** [comment]
I realized this is actually more likely configured within the [css language server](https://github.com/microsoft/vscode-css-languageservice/blob/bfb6a187a3c38a30e5a6e8ce5145c03a06cf233a/docs/customData.md), not here!

**Comment by jpnelson** [comment]
(and that there's a PR to add it: https://github.com/microsoft/vscode-css-languageservice/pull/276 )
